lib/s3.go

```diff
@@ -45,14 +45,14 @@ func (f *s3File) ETag() string {
 }
 
 func (f *s3File) Size() int64 {
-	return f.o.Size
+	return *f.o.Size
```

Comment:
Why?

Reply:
Thank you :)

I ran into build errors here where the size was changed from a value to a pointer after the dependency upgrades, hence the dereferencing.